2023/gaussian-convolution/spanish/sentence_translations.json

```
@@ -20,7 +20,7 @@
  {
   "input": "Of all the expressions we could dream up that give you some symmetric smooth graph with mass concentrated towards the middle, why is it that the theory of probability seems to have a special place in its heart for this particular expression?",
   "model": "nmt",
-  "translatedText": "De todas las expresiones que podríamos imaginar que dan una gráfica simétrica suave con masa concentrada hacia el centro, ¿por qué la teoría de la probabilidad parece tener un lugar especial en su corazón para esta expresión en particular?",
+  "translatedText": "De todas las expresiones que podríamos imaginar, que darían una gráfica simétrica, suave y con masa concentrada hacia el centro, ¿por qué la teoría de la probabilidad pareciera tener un lugar especial en su corazón para esta expresión?",
```

```suggestion
  "translatedText": "De todas las expresiones que podríamos imaginar, que darían una gráfica simétrica, suave y con masa concentrada hacia el centro, ¿por qué la teoría de la probabilidad parece tener un lugar especial en su corazón para esta expresión?",
```
It's more in line with the tense used in the English version and to me it seems more natural. What do you think?

---

Regardless, I think the phrase:

`¿Por qué la teoría de la probabilidad (parece | pareciera) tener un lugar especial en su corazón para esta expresión?`

Is not at all how Grant would have said it had he been a Spanish speaker anyways.

---

What do you propose then?

---

I suppose, something like this is better:
```suggestion
  "translatedText": "De todas las expresiones que podríamos imaginar que darían una gráfica simétrica, suave y con masa concentrada hacia el centro, ¿por qué parece que la teoría de la probabilidad tiene un lugar especial en su corazón para esta expresión?",
```

---

Yeah, it's way better !